fa25-team-b/cleaners/current_residence_name_normalization_v1.py

```
@@ -0,0 +1,86 @@
+Sources:
+    https://www.mass.gov/files/csv/2024-07/City_and_Town_Clerks.csv
+
+EDITS:
+    kristenbestavros - wrapped Brendan's work in a function so it can be called externally.
+'''
+#%% Data Import
+import pandas as pd
+
+# === ADDED: importable function wrapper ===
+def normalize_current_residence(df, city_index_csv: str):
+    import pandas as pd
```

Ruff (0.14.3)

Redefinition of unused `pd` from line 22

Remove definition: `pd`

(F811)